tests/integration/test_time_series.py

```diff
@@ -3,25 +3,23 @@
 import pathlib
```

Comment:
I've disabled all the tests in this file since we dropped shift for now.

Reply:
Yup and no intention of bringing back shift cause it makes assumptions about variables being stored as dataframes (instead of functions).

Reply:
Ah okay then we might as well remove the file completely soon.